packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -129,9 +129,9 @@ export const run = async () => {
 
   const clearPendingTxs = requiredEnvVars.CLEAR_PENDING_TXS === 'true'
 
-  const l1Provider = new JsonRpcProvider(requiredEnvVars.L1_NODE_WEB3_URL)
+  const l1Provider = new providers.JsonRpcProvider(requiredEnvVars.L1_NODE_WEB3_URL)
   const l2Provider = injectL2Context(
-    new JsonRpcProvider(requiredEnvVars.L2_NODE_WEB3_URL)
+    new providers.JsonRpcProvider(requiredEnvVars.L2_NODE_WEB3_URL)
```

Comment:
doesn't need to be in this PR but seems like we may be able to deprecate usage of `@ethersproject/providers` across packages